### PR TITLE
django wagtail cookiecutter

### PR DESCRIPTION
**Pull request by tkjone**
Fixed many of the issues from my last attempt to get this on the list:

- tests are working
- documentation links correct

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -197,6 +197,7 @@ Python-Django
 * `wagtail-cookiecutter-foundation`_: A complete template for Wagtail CMS projects featuring Zurb Foundation 5, ansible provisioning and deployment , front-end dependency management with bower, modular apps to get your site up and running including photo_gallery, RSS feed etc.
 * `django-starter`_: A Django template complete with vagrant and provisioning scripts - inspired by 12 factor apps and cookiecutter-django.
 * `cookiecutter-django-gulp`_: A Cookiecutter template for integrating frontend development tools in Django projects.
+* `wagtail-starter-kit`_: A cookiecutter complete with wagtail, django layout, vagrant, provisioning scrips, front end build system and more!
```
Comment (michaeljoseph) [inline README.rst]: provisioning _scripts_

**Comment by hackebrot** [comment]
:bell:

**Comment by hackebrot** [comment]
Can you please fix the typo that @michaeljoseph spotted? Thanks 🙇

**Comment by audreyr** [comment]
Can merge after https://github.com/tkjone/wagtail-starter-kit/pull/3 is merged.

Thanks for submitting this @tkjone!

**Pull request merged**